s/speexdsp/stone.yaml

```diff
@@ -0,0 +1,25 @@
+#
+# SPDX-FileCopyrightText: © 2020-2025 Serpent OS Developers
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+name        : speexdsp
```

Comment:
Perhaps this should just be `speex`?

Reply:
speex has multiple different libraries so naming it just speex doesn't make sense

Reply:
this is specially the dsp library and shouldn't be confused with speex